### ISSUE TITLE
How do I install .NET 4.8?

### ISSUE THREAD

**Issue by RUvlad1** [Skulltrail192/One-Core-API-Binaries#73]
When I launch the offline installer with the compatibility set to Windows 7, I get an error that says .NET can't be installed when compatibility options are set and that my OS isn't supported.

![Microsoft  NET Framework_b1](https://user-images.githubusercontent.com/60317461/177012805-31783d8c-7dc3-4a84-b74a-da7b8b88eca7.png)

**Comment by jonm58** [comment]
Run the offline installer and find the setup .exe in the extract folder to set the compatibility

**Comment by Legendary-Candice-Joe** [comment]
Might not be a great idea. When I installed .net 4.8, I got an error saying the program had to be terminated when ever I ran a .net program. I ended up using system restore to get rid of 4.8 because using the add or remove programs thing would crash it.

Could just me having bad luck with my install... but this happened on all one core api versions, on all of my installs.

**Comment by Skulltrail192** [comment]
The compatibility was bricked on version 2.5 of one-core-api. Now, it was fixed. However, i recommend use a alternative/repacked version of installer, avaliable on this forum: https://www.wincert.net/forum/topic/13805-microsoft-net-framework-472-full-x86x64-incl-language-packs-by-ricktendo/#comment-123251; You see 4.7.2 version, however, can search other version on forum. You can do your version, using tools on this page: https://github.com/abbodi1406/dotNetFx4xW7. These installers don't need set compatbility mode.

**Comment by RUvlad1** [comment]
> https://www.wincert.net/forum/topic/13805-microsoft-net-framework-472-full-x86x64-incl-language-packs-by-ricktendo/#comment-123251
the download link seems to be dead
![image](https://user-images.githubusercontent.com/60317461/177035877-6f739cd5-2354-420c-93c2-829481d38415.png)

I'm trying to figure out how to use the tools on the 2nd link you sent rn

**Comment by Skulltrail192** [comment]
Well, search on forum other links...I will upload my versions, but, you can did your version https://github.com/abbodi1406/dotNetFx4xW7. It's works with every .Net Framework 4.0+ versions;

**Comment by Legendary-Candice-Joe** [comment]
I used the github to make the dotnet installer, I'm still getting the "the program had to terminated error" (with a memory address). I've only been able to use dotnet 4.0, anything more keeps giving me the error.

**Comment by jonm58** [comment]
https://github.com/abbodi1406/dotNetFx4xW7 Useful
![1111](https://user-images.githubusercontent.com/73036321/177059340-28dc4ecd-e6ed-4072-9c8d-7b9d9973aec6.png)

**Comment by jonm58** [comment]
https://github.com/MishaTY/AuroraScreenSaver Working
![222](https://user-images.githubusercontent.com/73036321/177059424-da7a3f02-a2c2-4868-9c53-0a5bed532dc7.png)

**Comment by Legendary-Candice-Joe** [comment]
I guess I'll just mess around with a few things and see if I can get .net 4.8 to work.

it would be helpful if I can get a full tutorial thing?? Cause I used the https://github.com/abbodi1406/dotNetFx4xW7 and I was still getting the termated error. 

Technically this doesn't matter that much, especially since .net 4.5+ wasn't meant to run on windows xp. But it seems like everyone else isn't having the same trouble.

**Comment by hlizard** [comment]
The last time I tried OCA v2.5.0 on XP x64, I installed Net 4.8, I directly unzipped the official offline installation package "ndp48-x86-x64-allos-enu.exe" with 7-Zip, and then I tried to directly install the "netfx_full_x64.msi" in it. There were no restrictions, and it was directly successful. Then I tried several programs. WinForm and Gtk# programs ran, but WPF program could not run. The problem may be related to the exception "no imaging component suitable to complete this operation was found"

**Comment by Legendary-Candice-Joe** [comment]
I actually already tried that. I pretty much tried every version of dotnet 4.5 and 4.8, like the offline installers, the ones that have like weirder names.
 
Idk but I'll try removing dotnet 4.0, and installing every dotnet thing I can get. Also I did install vs express 2010 and although I got the "no imaging component suitable to complete this operation was found" before, this installation doesn't even have that. I could provide screenshots of it but before I give up entirely, I'm gonna try a few things first. If it doesn't work then ima call quits on .net 4.5+
